kolibri/plugins/user_profile/tasks.py

```diff
@@ -106,16 +108,33 @@ def mergeuser(command, **kwargs):
 
     local_user_id = kwargs.pop("local_user_id")
     local_user = FacilityUser.objects.get(id=local_user_id)
+    job = get_current_job()
+
     # Sync with the server to get the remote user:
     kwargs["no_push"] = True
-    call_command(command, **kwargs)
+    try:
+        call_command(command, **kwargs)
+    except MorangoError:
+        # error syncing with the server, probably a networking issue
+        job.storage.mark_job_as_failed(job.job_id, MorangoError, "")
```

Comment:
Did just letting the error raise here not work? The task runner should be catching the error from executing and then serializing/storing it on the job.

Reply:
ah, no, I assumed it was necessary. Just checked and the error message is slightly different, but useful anyway, so I'm removing this line.

Reply:
If you want to customize the error message, you can reraise the error with a different message if needed!